### PR TITLE
Docs: minor corrections to prefix and naming

### PR DESCRIPTION
**Pull request by brendamuir**
Adds correct prefix for mimir 
Corrects notifiers to contact point types terminology

### CODE REVIEW

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ You can create and manage recording rules for an external Grafana Mimir or Loki
 
   - **Loki** - The `local` rule storage type, default for the Loki data source, supports only viewing of rules. To edit rules, configure one of the other rule storage types.
 
-  - **Grafana Mimir** - use the [legacy `/api/prom` prefix](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#path-prefixes), not `/prometheus`. The Prometheus data source supports both Grafana Mimir and Prometheus, and Grafana expects that both the [Query API](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#querier--query-frontend) and [Ruler API](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#ruler) are under the same URL. You cannot provide a separate URL for the Ruler API.
+  - **Grafana Mimir** - use the `/prometheus` prefix. The Prometheus data source supports both Grafana Mimir and Prometheus, and Grafana expects that both the [Query API](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#querier--query-frontend) and [Ruler API](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#ruler) are under the same URL. You cannot provide a separate URL for the Ruler API.
```
Comment (osg-grafana) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:30]: @pracucci, can you vet this for accuracy?

**Review by osg-grafana — APPROVED** [review]
Looks good to me, but requesting engineering input

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md (hunk 1 above)
Comment (pracucci) [reply docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:30]: Correct.

**Review by pracucci — APPROVED** [review]

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md (hunk 1 above)
Comment (brendamuir) [reply docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md:30]: It has engineering input -- it was approved by Dimitar :)

**Comment by osg-grafana** [comment]
@dimitarvdimitrov no-op, just linking you as a technical reviewer explicitly.

**Pull request merged**